Index: frontend/src/shiftservice-client/model/notification-settings-dto.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/frontend/src/shiftservice-client/model/notification-settings-dto.ts b/frontend/src/shiftservice-client/model/notification-settings-dto.ts
--- a/frontend/src/shiftservice-client/model/notification-settings-dto.ts
+++ b/frontend/src/shiftservice-client/model/notification-settings-dto.ts	(date 1765919986585)
@@ -10,7 +10,7 @@
 
 
 export interface NotificationSettingsDto { 
-    perTypeSettings: { [key: string]: Set<InnerEnum>; };
+    perTypeSettings: { [key: string]: Set<NotificationSettingsDto.PerTypeSettingsEnum>; };
 }
 export namespace NotificationSettingsDto {
     export const PerTypeSettingsEnum = {
